var/spack/repos/builtin/packages/lfortran/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Lfortran(CMakePackage):
+    """Modern interactive LLVM-based Fortran compiler"""
+
+    homepage = "https://lfortran.org"
+
+    git = "https://gitlab.com/lfortran/lfortran.git"
+
+    version("master", branch="master")
+    version("0.16.0", tag="v0.16.0")
+    version("0.15.0", tag="v0.15.0")
+
+    depends_on("python@3:", type="build")
+    depends_on("cmake", type="build")
+    depends_on("llvm@14.0.4", type="build")
+    depends_on("zlib@1.2.12", type="build")
+    depends_on("re2c@2.2", type="build")
+    depends_on("bison@3.4.2", type="build")
+    depends_on("binutils@2.38", type="build", when="platform=linux")
+
+    maintainers = ["certik"]
+
+    @run_before("cmake")
+    def prepare_source(self):
+        open("version", "w").write(str(self.version))
+
+        python = which("python")
```

Comment:
```suggestion
        python = self.spec["python'].command
```
This will eliminate any issue of `python` vs `python3`